db/db_impl/db_impl_open.cc

```diff
@@ -1755,13 +1758,12 @@ Status DBImpl::Open(const DBOptions& db_options, const std::string& dbname,
         FileType file_type;
         std::string file_path = path + "/" + file_name;
         if (ParseFileName(file_name, &file_number, &file_type) &&
-            file_type == kTableFile) {
+            (file_type == kTableFile || file_type == kBlobFile)) {
```

Comment:
I think we should probably leave this as-is for now (i.e. don't use this branch for blob files) since we know blob files are not there in the map at the moment. (In other words, this should be updated when the `TODO` above is addressed.)

Reply:
Can you explain why the blob files won't be there in map?  It gets all the existing files by calling env->GetChildren.

Reply:
Right, sorry, I misread the code